result/dhcpv6server.yaml

```diff
@@ -0,0 +1,161 @@
+      type: object
+      properties:
+        server_names:
+          description: >-
+            The names of DHCPv6 Servers to return results for.
+            An empty list will return results for all DHCPv6 Server.
+          type: array
+          items:
+            type: string
+          x-constraint:
+          - "/components/schemas/Device.Dhcpv6Server/properties/name"
+          x-field-uid: 1
+        column_names:
+          description: >-
+            The list of column names that the returned result set will contain.
+            If the list is empty then all columns will be returned except for
+            any result_groups. The name of the DHCPv6 server cannot be excluded.
+          type: array
+          items:
+            type: string
+            x-enum:
+              solicits_received:
+                x-field-uid: 1
+              solicits_ignored:
+                x-field-uid: 2
+              advertisements_sent:
+                x-field-uid: 3
+              requests_received:
+                x-field-uid: 4
+              requests_naked:
+                x-field-uid: 5
+              confirms_received:
+                x-field-uid: 6
+              renewals_received:
+                x-field-uid: 7
+              rebinds_received:
+                x-field-uid: 8
+              replies_sent:
+                x-field-uid: 9
+              releases_received:
+                x-field-uid: 10
+              declines_received:
+                x-field-uid: 11
+              information_requests_received:
+                x-field-uid: 12
+              relay_forwards_received:
+                x-field-uid: 13
+              relay_replies_sent:
+                x-field-uid: 14
+              reconfigures_sent:
+                x-field-uid: 15
+          x-field-uid: 2
+    Dhcpv6Server.Metric:
+      description: >-
+        DHCPv6 per server statistics information.
+      type: object
+      properties:
+        name:
+          description: >-
+            The name of a configured DHCPv6 Server.
+          type: string
+          x-field-uid: 1
+        solicits_received:
+          description: >-
+            Number of DHCPSOLICIT messages received.
+          type: integer
+          format: uint64
+          x-field-uid: 2
+        solicits_ignored:
+          description: >-
+            Number of DHCPSOLICIT messages ignored.
+          type: integer
+          format: uint64
+          x-field-uid: 3
+        advertisements_sent:
+          description: >-
+            Number of DHCP Advertise messages send.
+          type: integer
+          format: uint64
+          x-field-uid: 4
+        requests_received:
+          description: >-
+            Number of DHCPREQUEST messages received.
+          type: integer
+          format: uint64
+          x-field-uid: 5
+        requests_naked:
+          description: >-
+            Number of DHCPREQUEST messages naked.
```

Comment:
Probably should be "Number of negative lease DHCPNACK messages sent." to match with DHCPv4 server and v6 side documentation.